docs/post-v1-determineversion.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: page
+title: Experimental determineversion
+permalink: /post-v1-determineversion/
+parent: API
+nav_order: 5
+---
+# POST /v1experimental/determineversion
+Experimental
+{: .label }
+
+Given the source code of C/C++ libraries, this endpoint attempts to find the closest library and version.
+
+The purpose of the endpoint is to help determine the version of a given C/C++ library. It is difficult to know the correct version of C/C++ projects because there is not a centralized package manager within the ecosystem. This API endpoint helps bridge that gap. Once you have the likely version, you can use [POST v1/query](post-v1-query.md) or [POST v1/queryset](post-v1-queryset.md) to search for vulnerabilities. 
```

Comment:
Can we provide some info on which repos we index? And our future plans to extend coverage?

Reply:
Added. Thanks!